docs/content/doc/administration/config-cheat-sheet.en-us.md

```diff
@@ -283,6 +283,7 @@ The following configuration set `Content-Type: application/vnd.android.package-a
 
 - `APP_DATA_PATH`: **_`AppWorkPath`_/data**: This is the default root path for storing data.
 - `PROTOCOL`: **http**: \[http, https, fcgi, http+unix, fcgi+unix\]
+  - Note: Value is case-sensitive (must be in lowercase). For example, if you write `PROTOCOL=HTTPS`, you may get a SSL_ERROR_RX_RECORD_TOO_LONG error on Firefox or ERR_SSL_PROTOCOL_ERROR on Chrome and Edge.
```

Comment:
Hmm… I'm not sure if we need this change.
There are quite a few options that explicitly say they only support certain values.
Basically all of them are case-sensitive.
As an example, this is line 220:
```md
- `DEFAULT_THEME`: **auto**: \[auto, gitea, arc-green\]: Set the default theme for the Gitea install.
```
should we document for all of them that they are case-sensitive?
Plus, if you update it here, you must also update `custom/conf/app.example.ini`.

Reply:
We should ideally just make it case-insensitive if the meaning is still clear, which is the case for `PROTOCOL` at least.

Reply:
I do not think it is good to add "if you write PROTOCOL=HTTPS, ....." into document either.